lib/openfoodfacts.dart

```diff
@@ -855,4 +856,33 @@ class OpenFoodAPIClient {
       return KnowledgePanels.empty();
     }
   }
+
+  /// Returns the nutrient hierarchy specific to a country, localized.
+  ///
+  /// [cc] is the country code, as ISO 3166-1 alpha-2
+  static Future<OrderedNutrients?> getOrderedNutrients({
+    required final String cc,
+    required final OpenFoodFactsLanguage language,
```

Comment:
In first approach I don't think it would make sense to support several languages at the same time: here we're talking about nutrients, not localized food or ingredient names where we would need fallbacks.

Reply:
Okay that sense